src/MediaServiceProvider.php

```diff
@@ -10,11 +10,25 @@
 use DrewRoberts\Media\Policies\ImagePolicy;
 use DrewRoberts\Media\Policies\TagPolicy;
 use DrewRoberts\Media\Policies\VideoPolicy;
+use Illuminate\Support\Str;
 use Tipoff\Support\TipoffPackage;
 use Tipoff\Support\TipoffServiceProvider;
 
 class MediaServiceProvider extends TipoffServiceProvider
 {
+    public function boot()
+    {
+        parent::boot();
+
+        Str::macro('keepAlphanumericCharacters', static function ($value) {
+            return preg_replace('/[^\w\s]/', '', $value);
+        });
+
+        Str::macro('splitAtCapitalLetters', static function ($value) {
+            return implode(' ', preg_split('/(?=[A-Z])/', $value));
+        });
```

Comment:
I don't think we need this. When running it with a space through the slug, then it would add a dash and I don't want dashes in tag slugs. For example, no "tags/drew-roberts" and instead use just "drewroberts" as the slug so the path is "tags/drewroberts"

Reply:
Done!